policy/datasets.yaml

```diff
@@ -980,4 +980,43 @@ dcrDiff_kernelSrc:
     persistable:   SourceCatalog
     storage:       FitsCatalogStorage
     tables:        'raw'
-
+makeBrighterFatterKernel_config:
+    description: "Configuration for the brighter-fatter coefficient calculation code."
+    persistable: Config
+    python: lsst.cp.pipe.brighterFatterKernelGeneration.MakeBrighterFatterKernelTaskConfig
+    storage: ConfigStorage
+    tables:
+    - raw
+    - raw_visit
+    template: config/makeBrighterFatterKernel.py
+makeBrighterFatterKernel_metadata:
+    description: "Metadata for the brighter-fatter coefficient calculation code."
+    persistable: PropertySet
+    storage: BoostStorage
+    python: lsst.daf.base.PropertySet
+    tables: raw
+    template: brighterFatter_metadata/brighterFatter_metadata.boost
+brighterFatterKernelNew:
```

Comment:
Do you need the "New" here? I thought the old name that HSC used was "bfKernel" and that "brighterFatterKernel" had not been used before.

Reply:
Ack. I agree that it has not been used, and could be used in that way in principle. However, I think that while two things coexist, it is nice to have the added disambiguation of the "new" on the end, as things are confusing enough already, plus this saves having to change it. I think that once the old one is deliberately removed that we then update the name so that the One True Kernel be {{brighterFatterKernel}} as you suggest. Does that sound OK for now?

Reply:
I worry that it is disruptive to have to change the name of a dataset. However, as long as the dataset name is purely internal (does not affect the name or path of persisted data) then it's probably not too bad. Go ahead.

If the dataset name affects the path or file name of the persisted data then I would be very uncomfortable changing the name later.

Reply:
Well, our OOB discussion persuaded me to just change it. Doing so now :)